### DISCUSSION TITLE
Read replica for the flapdoodle instance(for change stream usage)

### DISCUSSION THREAD

**Discussion by roieBeckIntuit** [flapdoodle-oss/de.flapdoodle.embed.mongo#483] [Q&A]
Hi,
Didn't find any guide on how to achieve it in 4.9.2,
I would like to run the new version with read replica in order to use change stream,
I looked but didn't found any guide here:
https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/blob/main/docs/Howto.md
can you please help with an example on how to add a reading replica instance?
Many thanks

**Comment by michaelmosmann** [comment]
@roieBeckIntuit do you have some link which describes your use case?

**Comment by bodote** [comment]
We have a working solution , see : https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/discussions/496 although its not perfect.
